R-packages/covidcast/R/covidcast.R

```diff
@@ -761,8 +764,8 @@ covidcast <- function(data_source, signal, time_type, geo_type, time_values,
 .request <- function(params) {
   # API call. Allow base API URL to be replaced, e.g. to use a staging/testing
   # server when needed.
-  response <- httr::GET(getOption("covidcast.base_url", default = COVIDCAST_BASE_URL),
-                        httr::user_agent("covidcastR"), query = params)
+  response <- httr::POST(getOption("covidcast.base_url", default = COVIDCAST_BASE_URL),
+                        httr::user_agent("covidcastR"), body = params)
```

Comment:
[Sam raised](https://github.com/cmu-delphi/delphi-epidata/pull/309#issuecomment-740746189) that GET requests can be cached along the way, while POST request generally can't/aren't. This was handled in the python client by using GET by default, and switching to POST on 414. Is that relevant for httr calls?

Reply:
I would assume so, since it has to do with the type of request and not the origin. I'll change it.